examples/payjoin-0.13.0.md

```diff
@@ -0,0 +1,347 @@
+            )
+            .unwrap()
+            .extract_v1()
+            .unwrap();
+        // Step 5. Send the request and receive response
+        // let payjoin_url = pj_uri.extras.e
+        // BITCOIN:BCRT1Q0S724W239Z2XQGSZV6TE96HMYLEDCTX3GDFZEP?amount=0.01&pj=https://localhost:3000
+        //
+        let agent = https_agent();
+        let res = agent
+            .post(req.url.as_str()) // pj_uri.endpoint
+            .set("Content-Type", "text/plain")
+            .send_string(&String::from_utf8(req.body).unwrap())
+            .unwrap();
+        // Step 6. Process the response
+        //
+        // An `Ok` response should include a Payjoin Proposal PSBT.
+        // Check that it's signed, following protocol, not trying to steal or otherwise error.
+        let psbt = ctx
+            .process_response(&mut res.into_string().unwrap().as_bytes())
+            .unwrap();
+        // Step 7. Sign and finalize the Payjoin Proposal PSBT
+        //
+        // Most software can handle adding the last signatures to a PSBT without issue.
+        let psbt = sender_wallet
+            .wallet_process_psbt(&base64::encode(psbt.serialize()), None, None, None)
+            .unwrap()
+            .psbt;
+        let tx = sender_wallet
+            .finalize_psbt(&psbt, Some(true))
+            .unwrap()
+            .hex
+            .unwrap();
+        // Step 8. Broadcast the Payjoin Transaction
+        let txid = sender_wallet.send_raw_transaction(&tx).unwrap();
+        txid
+    }
+}
+
+fn main() {
+    setup_wallets();
+    let args: Vec<String> = std::env::args().collect();
+    let payjoin_uri: String = args[1].parse().unwrap();
+    let payjoin_uri = payjoin::Uri::try_from(payjoin_uri)
+        .unwrap()
+        .assume_checked();
+    let txid = Sender::try_payjoin(payjoin_uri);
+    dbg!(&txid);
+}
+
+  ```
+
+# Setup payjoin-cli
+ Setup `payjoin-cli` and start the receiver side so we can test the
+ sender implementation.
+
+ 0. Open new shell window
+ 1. Install `payjoin-cli`
+
+ ```shell
+ cargo install payjoin-cli --version 0.0.2-alpha
+ ```
+ 2. Run payjoin-cli as the receiver
+ ```rust
+  RUST_LOG=debug payjoin-cli --features=danger-local-https -- -r http://localhost:18443/wallet/reciever --rpcuser foo --rpcpass qDDZdeQ5vw9XXFeVnXT4PZ--tGN2xNjjR4nrtyszZx0= receive --endpoint https://localhost:3000 1230000000
+ ```
+
+ Once you run the the above command, an http server will start and a
+ payjoin uri will be printed to the stdout, and will look like this:
+
+ ```shell
+ bitcoin:bcrt1qstujjzput7eznwv27e7qh7lpkgz33fh39g2gpt?amount=12.3&pj=https://localhost:3000/&pjos=0
+ ```
+
+ Copy the payjoin uri as we will need it in the next(final) step.
+
+# Run
+
+  Lets `cd` back to the `payjoin-tutorial` folder and let the sender
+  make a request to the payjoin uri generated in the previous step.
+
+  ```shell
+  cargo run -- "bitcoin:bcrt1qstujjzput7eznwv27e7qh7lpkgz33fh39g2gpt?amount=12.3&pj=https://localhost:3000/&pjos=0"
```

Comment:
this will return txid, it would be nice if we wrote a couple of more steps to try and fetch the tx from the node and explain how the inputs and outputs were distributed and how payjoin played a role.